examples/basic/docker-compose.yml

```
@@ -26,11 +26,11 @@ services:
       - default
 
   mautic_web:
-    image: mautic/mautic:5-apache
+    image: mautic/mautic:latest
     links:
       - db:mysql
     ports:
-      - 8001:80
+      - 8080:80
```

What happens when someone is already running another server using 8080? For example, they might be running their website locally on this port I guess, it's quite a common one. Would it make a problem if this was set to 8080?

---

In this scenario, the Docker Compose setup would fail and flag the port as being occupied.

For production environments, I would typically expect users to set up a reverse proxy to manage multiple services on the same port. The reverse proxy would also handle SSL termination, which our container doesn’t do.

For demo purposes, I expect that users will use our default Compose configuration as-is. However, it might be a good idea to change the default port to something less conventional, like 1111. This could serve two purposes:

- Avoid conflicts, as you said

- Improve security, as a "weird" port can help prevent easy discovery of the service on a poorly configured firewall in front of the machine, as no widely known services typically run on this port.

Would you agree to
```suggestion
      - 1111:80
```
?